lib/swaps/Swaps.ts

```diff
@@ -1354,6 +1354,7 @@ class Swaps extends EventEmitter {
         peerPubKey: deal.peerPubKey,
         role: deal.role,
       };
+      this.logger.info(`Successfully executed swap ${deal.rHash} with peer ${deal.peerPubKey} (${this.pool.getPeer(deal.peerPubKey).alias})`);
```

Comment:
This may be minor but I noticed that we're using `getPeer` here which could fail if we've disconnected from the peer. It's somewhat of an edge case but it could happen. Rather than risk hitting an unhandled exception here, I'd do `this.pool.getNodeAlias(deal.peerPubKey)` instead to get the alias which will return a value even if we're no longer connected to our swap partner.